README.rst

```diff
@@ -201,6 +201,16 @@ This entry may be present if you named your server with the LMS hostname.
 
 .. _DigitalOcean: https://digitalocean.com/
 
+
+Use event-bus through tutor-discovery
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Event bus can be run through discovery. To consume events from event bus in discovery, follow these `instructions`_ provided by `event-bus-redis`_.
```

Comment:
We can mention that tutor-discovery does not start event bus consumption by default but supports running it. And if someone wants to, they can do so by following the doc/link.